android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIProgressIndicator.java

```diff
@@ -109,10 +75,15 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 
 		} else if (key.equals(TiC.PROPERTY_VALUE)) {
 			if (visible) {
-				int value = TiConvert.toInt(newValue);
-				int thePos = (value - min) * incrementFactor;
-
-				handler.obtainMessage(MSG_PROGRESS, thePos, -1).sendToTarget();
+				int progressValue = (TiConvert.toInt(newValue) - this.min) * this.incrementFactor;
```

Comment:
Set default to zero?
```Java
TiConvert.toInt(newValue, 0)
```

Reply:
Yeah, I'll do that. Especially since we're doing this when opening the dialog too.